coremltools/converters/mil/frontend/torch/ops.py

```diff
@@ -4525,6 +4549,25 @@ def arange(context, node):
     context.add(res)
 
 
+@register_torch_op(torch_alias=["arange.start_step"])
+def arange_start_step(context, node):
+    inputs = _get_inputs(context, node)
+    start = inputs[0]
+    end = inputs[1]
+    step = 1 if len(inputs) < 3 else inputs[2]
+
+    int_start = isinstance(start, int) or types.is_int(start.dtype)
```

Comment:
I think it would be a lot clearer if you renamed `int_start` to `start_is_int`. I would make the same change for `int_end` and `int_step`.

Reply:
revised as suggested